init.lua

```diff
@@ -112,19 +114,39 @@ local KILL_OK = false
 local AREA_ACTIVATE = KILL_OK
 local AREA_NOACTIVATE = KILL_NO
 
+local savemodeToString = "FALSCH" -- only for debugging
 if safemode then
 	AREA_ACTIVATE = KILL_NO
 	AREA_NOACTIVATE = KILL_OK
+	savemodeToString = "WAHR"
```

Comment:
What is this? I cannot merge this.

Also, it uses bad camel case - did you mean `saveModeToString` ? `safeModeToString` ? what is it meant to do?

Also, do not use camel case for local variables, use all-lower-case like, and optionally use underscores like the existing local variables. Or if it is meant to be a global variable, use all upper case.